treebeard/mp_tree.py

```diff
@@ -373,8 +373,11 @@ def process(self):
                       ' increasing the path.max_length property'
                       ' and UPDATE your database'))
         else:
-            # adding the new child as the last one
-            newobj.path = self.node.get_last_child()._inc_path()
+            # adding the new child as the last one after checking if there is a Last Child
+            if self.node.get_last_child():
+                newobj.path = self.node.get_last_child()._inc_path()
+            else:
+                newobj.path = self.node.path + '0001'
```

Comment:
```suggestion
            last_child = self.node.get_last_child():
            if last_child is not None:
                newobj.path = last_child._inc_path()
            else:
                newobj.path = self.node.path + '0001'
```

Reply:
but you can actually just use the absence of a child node to determine if a node is a leaf node: https://github.com/django-treebeard/django-treebeard/pull/235/files#diff-35501ef525349cd39e4713d1a9f64a249fa4fbd31d875513e3a15e65988701a2R365